.github/workflows/release-wc-beta-tester.yml

```diff
@@ -0,0 +1,33 @@
+name: WooCommerce Beta Tester Release
+
+on:
+    workflow_dispatch:
+        inputs:
+            version:
+                description: 'The version number for the release'
+                required: true
+
```

Comment:
Would be a good idea to lock this workflow down initially with `permissions: {}` which gives zero permissions. Then unlock the minimum needed permissions in the jobs level which you have already done.

Reply:
Good point! addressed in https://github.com/woocommerce/woocommerce/pull/36387/commits/ed342a78918bb65d0f53ed4c514b0e1185c8d57c